.github/workflows/release.yaml

```diff
@@ -40,3 +46,13 @@ jobs:
           # GitHub sets the GITHUB_TOKEN secret automatically.
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           GPG_FINGERPRINT: ${{ steps.import_gpg.outputs.fingerprint }}
+
+      - name: Release new version to spacelift
+        if: startsWith(github.ref, 'refs/tags/')
+        env:
+          GPG_KEY_ID: ${{ steps.import_gpg.outputs.keyid }}
+          SPACELIFT_API_KEY_ENDPOINT: https://spacelift.ktl.net
+          SPACELIFT_API_KEY_ID: ${{ secrets.SPACELIFT_API_KEY_ID }}
+          SPACELIFT_API_KEY_SECRET: ${{ secrets.SPACELIFT_API_KEY_SECRET }}
+        run:
+          spacectl provider create-version --type=nexus
```

Comment:
do we also need to add `spacectl provider publish-version`

Reply:
Yes.